PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/IntelliJAzureIcons.java

```diff
@@ -131,7 +131,13 @@ public static Icon getIcon(@Nonnull AzureIcon azureIcon) {
         return azureIcons.computeIfAbsent(azureIcon, IntelliJAzureIcons::getAzureIcon);
     }
 
+    @Nullable
     private static Icon getAzureIcon(@Nonnull AzureIcon azureIcon) {
-        return doGetIcon(AzureIcon.getIconPathWithModifier(azureIcon), azureIcon.getIconPath(), IntelliJAzureIcons.class);
+        final String pathWithModifier = AzureIcon.getIconPathWithModifier(azureIcon);
```

Comment:
the logic should be check path with modifier, if null, fall back to use path, so it should be ok to have pathWithModifier is null